assets/app/features/upgrade/templates/landing-page.jade

```diff
@@ -11,3 +11,4 @@
     .row.hidden-md.hidden-lg.step-3(ng-class="{'enabled': upgradeLandingVm.mode.valid}")
         .col-md-6
             include landing-page_step3.partial.jade
+suse-modal(error="upgradeLandingVm.error")
```

Comment:
The idea is that the directive will encapsulate the modal. The controller will be just used to bind the directive with the error source. Controllers should not create/delete view elements.